client/src/app/site/pages/meetings/pages/agenda/modules/topics/modules/topic-poll/components/topic-poll-vote/topic-poll-vote.component.html

```diff
@@ -173,3 +178,19 @@ <h4 *ngIf="(poll.isMethodY || poll.isMethodN) && poll.max_votes_amount > 1">
         </button>
     </div>
 </ng-template>
+
+<ng-template #sendAll let-delegation="delegations">
+    <div class="centered-button-wrapper margin-top" *ngIf="!canStillVote()">
+        <button
+            mat-flat-button
+            color="accent"
+            (click)="submitVotes(delegations)"
```

Comment:
Why only the delegations? Why are you even using them as a parameter for this `submitVotes` method? They're saved in the component, right? No need to pass them through the template, just get them within the method.